src/App.js

```diff
@@ -1,57 +1,144 @@
-import React from 'react';
-import Homepage from './Homepage';
-import './App.css';
+import React, { useState, useEffect } from "react";
+import Homepage from "./Homepage";
+import "./App.css";
+import { firstHomepage, secondHomepage, thirdHomepage } from "./homepages";
+import Login from "./login";
+import fire from "./fire.js";
+import Toppage from "./toppage.js";
```

Comment:
You don't need to add the extension when importing a JS file
```suggestion
import Toppage from "./toppage";
```